### DISCUSSION TITLE
problem with character making

### DISCUSSION THREAD

**Discussion by monkeystink** [Leather128/LeathersFunkinEngine#14] [Q&A] [answered]
ok so im using your engine and i want to add a character but it crashes my game everytime i try to insert it and yes im making it with tools in settings

**Comment by Leather128** [comment, marked as answer]
oh, you probably shouldn't make the character with the in-game tool yet (as it's unfinished)

instead just use another character as a reference (like bf or something)

also remember to make icons for character ( or else it no load :((((((((((((( )

**Comment by monkeystink** [comment]
ok
